HPC/Iridis4/subLotus

```diff
@@ -23,7 +24,7 @@ module load R/3.3.0
 # Change to directory from which job was submitted
 cd $PBS_O_WORKDIR
 
-# Run Lotus (mpif90). Note that total np must be the same as stated before. You can also use your /scratch
+# Run Lotus (mpif90). Note that total np must be the same as stated before. You can also use your /s$
```

Comment:
Same here, I guess this is a bash variable for scratch? This is just a comment though, so the more explicit the better ;)